package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "one-small-thing-fe",
-  "homepage": "https://one-small-thing.github.io/One-Small-Thing-FE/",
```

Comment:
I think we have to keep this for our testing

Reply:
@camianderson @LINTONBNEIL This is required for your automatic deployment to work using GitHub pages.